components/Common/ThemeToggle/__tests__/index.test.mjs

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { LocaleProvider } from '@/providers/localeProvider';
+
+import ThemeToggle from '../';
+
+let mockCurrentTheme = 'light';
+
+const toggleTheme = () => {
+  mockCurrentTheme = mockCurrentTheme === 'light' ? 'dark' : 'light';
+};
+
+function renderThemeToggle() {
+  return render(
```

Comment:
You can put `renderThemeToggle` inside a `beforeEach` callback. DRY

Reply:
Done !